backend/src/data/migrations/20220918074339_change_interview_status_to_enum.ts

```diff
@@ -0,0 +1,50 @@
+import { Knex } from 'knex';
+
+enum InterviewStatus {
+  PENDING = 'pending',
+  IN_PROGRESS = 'in-progress',
+  COMPLETED = 'completed',
+  REJECTED = 'rejected',
+  NEW = 'new',
+  CANCELED = 'canceled',
+}
+
+enum InterviewStatusUppercase {
+  IN_PROGRESS = 'In Progress',
+}
+
+const TABLE_NAME = 'interviews';
+
+enum ColumnName {
+  STATUS = 'status',
+  STATUS_CHECK_CONSTRAINT = 'interviews_status_check',
+}
+
+async function up(knex: Knex): Promise<void> {
+  await knex.schema.raw(`
+    UPDATE interviews SET status = (CASE WHEN status = '${InterviewStatusUppercase.IN_PROGRESS}' THEN '${InterviewStatus.IN_PROGRESS}'
+      ELSE LOWER(status)
+  END);`);
+
+  await knex.schema.raw(`
+    ALTER TABLE "${TABLE_NAME}"
+    ADD CONSTRAINT "${ColumnName.STATUS_CHECK_CONSTRAINT}"
+    CHECK (${ColumnName.STATUS} IN ('${Object.values(InterviewStatus)
+    .map((status) => `'${status}'`)
+    .join(', ')}'))
```

Comment:
Looks like this quotes are not needed 😕 (that proposals were separate, like one or another, not both together 😅)
```suggestion
    CHECK (${ColumnName.STATUS} IN (${Object.values(InterviewStatus)
    .map((status) => `'${status}'`)
    .join(', ')}))
```

Reply:
Oh right, how did I miss it :confused: